src/molecules/singleLineInput/component.tsx

```diff
@@ -77,7 +77,7 @@ export const SingleLineInput = (props: ISingleLineInputProps): React.ReactElemen
     }
   };
 
-  const onClick = (): void => {
+  const onClick = () => {
```

Comment:
this should really be onClicked (as should the prop). Can you please update this one and add a todo above the prop saying to change it to `onClicked`

Reply:
but we will have to change this in all other project where we use `singleInputField`

Reply:
dont change the prop, just this intenral function. put a TODO above the prop